ladybug/src/main/resources/springIbisTestTool.xml

```diff
@@ -86,23 +101,34 @@
 				'${dtap.stage}' == 'xxx' ||
 				'${dtap.stage}' == ''
 				?
-				'testStorageXml' : 'testStorageFile'
+				'testStorageXml' : 'testStorageDatabase'
```

Comment:
Willen we nooit meer kunnen kiezen voor storage in file?

Reply:
Als je een database hebt en de Ladybug tabel bestaat (al dan niet doordat de applicatie rechten heeft om dat met Liquibase te doen) dan is het het mooiste om die te gebruiken, zeker in een Docker/Kubernetes omgeving zodat je data niet telkens verloren gaat. Heb je geen database en Ladybug tabel beschikbaar dan doet hij fallback naar file storage, zie ProxyStorage hierboven en hieronder en https://github.com/wearefrank/ladybug/blob/master/src/main/java/nl/nn/testtool/storage/proxy/ProxyStorage.java#L35

Reply:
Dat had ik nog niet gezien!